ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/Solaris.yml

```diff
@@ -285,3 +285,43 @@
     state: link
     force: yes
   tags: build_tools
+
+- name: Download git 2.31.0 Source
+  command: wget https://mirrors.edge.kernel.org/pub/software/scm/git/git-2.31.0.tar.gz -O /tmp/git-2.31.0.tar.gz
+  tags:
```

Comment:
This isn't currently guarded with a "Do we have a suitable version installed" check so it will download and rebuild each time the playbooks are run, which is not idea. Somethign similar to the checks in the main [GIT_Source])https://github.com/adoptium/infrastructure/blob/master/ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/GIT_Source/tasks/main.yml) role should avoid that being a problem.

Reply:
Amendments to check sha completed.